languages/en.ini

```diff
@@ -1610,6 +1624,7 @@ View Retraction Notice = "View Retraction Notice"
 View this record in EBSCOhost = "View this record in EBSCOhost"
 view_already_selected = "%%current%% view already selected"
 view_in_opac = "View in OPAC"
+viewing_request = "Request Viewing"
```

Comment:
I'm not sure if "Viewing" is the clearest word here. Would "Request Access" be better? (Though I understand that the meaning is a little different -- maybe you're trying to express something more specific than just "access").